pycircleci/api.py

```diff
@@ -336,6 +336,24 @@ def cancel_build(self, username, project, build_num, vcs_type=GITHUB):
         resp = self._request(POST, endpoint)
         return resp
 
+    def get_pipelines(self, username, project, vcs_type=GITHUB):
```

Comment:
I'd suggest using the `slug` arg, instead of the tuple (username, project, vcs_type),
similar to https://github.com/alpinweis/pycircleci/blob/master/pycircleci/api.py#L63